README.md

```diff
@@ -11,6 +11,83 @@ An OpenAI-powered triage bot for a slack support channel designed to tag oncalls
 
 ## Usage
 
+Triage-bot monitors your Slack support channels and automatically assists with user support requests. It integrates with your existing Slack workspace and requires minimal setup to get started.
+
+### Basic Workflow
+
+1. **User posts a message** in a Slack channel where triage-bot is active
+2. **Triage-bot analyzes the message** using LLMs to determine the nature and urgency
+3. **Bot takes appropriate actions**:
+   - Tags relevant on-call personnel
+   - Classifies the issue (Bug, Feature, Question, Incident, Other)
+   - Provides helpful context from past discussions or web searches
+   - Suggests potential solutions when confident
+
+### Supported Commands
+
+Users can interact directly with triage-bot by @-mentioning it:
+
+- **Help requests**: `@triage-bot why is my build failing?`
```

Comment:
Also note that top-level comments that don't tag the bot will also be responded to.